### ISSUE TITLE
Ace Combat Assault Horizon - Enhanced Edition (228400)

### ISSUE THREAD

**Issue by FischerDavid** [ValveSoftware/Proton#2160]
There were no other users online to test multiplayer functionality.  Single Player has been working well for me.  I did find an issue where the game crashes when attempting to view leaderboard information.

# Compatibility Report
- Name of the game with compatibility issues: Ace Combat Assault Horizon - Enhanced Edition
- Steam AppID of the game: 228400

## System Information
- GPU: Nvidia GTX 970
- Driver/LLVM version: NVIDIA 390.87
- Kernel version: 4.18.0-12-generic
- Link to full system information report as [Gist](https://gist.github.com/FischerDavid/d7632499b9b88187711f3b27eba151a3):
- Proton version: 3.16-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Game crashes when attempting to view leaderboard data
[steam-228400.log](https://github.com/ValveSoftware/Proton/files/2716972/steam-228400.log)

## Reproduction
Main Menu > Multiplayer > Leaderboards
Select any of the 3 leaderboard choices and the game crashs

**Comment by kisak-valve** [comment]
Ace Combat Assault Horizon - Enhanced Edition (228400)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7255.
@AlexanderWKoenig posted on 2023-11-13T16:49:58:

# Compatibility Report
- Name of the game with compatibility issues: Ace Combat Assault Horizon - Enhanced Edition
- Steam AppID of the game: 228400

## System Information
- GPU: AMD RX6800
- Video driver version: Mesa 23.2.1
- Kernel version: 6.5.8-273-tkg-linux-tkg-eevdf
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/AlexanderWKoenig/8712d2344c6471c7ddc312fe74e85196
- Proton version:  8.0-4c

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-228400.log](https://github.com/ValveSoftware/Proton/files/13337868/steam-228400.log)

## Symptoms
Game crashes my entire X11 session at start of campain mission 5.
Here you have to land the plan I suppose. After the intro cutscene the game runs for about 5 seconds and crashes.
I've also tried proton experimental and proton-ge but the same issue appears. 
This happens also on my SteamDeck. Here it wont crash the session, but screen goes black for a while, after this the game is unresponsive.

## Reproduction
Try to play the 5th campain mission and wait for it do crash. Should be about 30s after mission loads

**Comment by Blisto91** [comment]
@AlexanderWKoenig Hello and thank you for the report. 
Can you retest with Proton Experimental? There's made a fix that should hopefully have fixed it.

**Comment by AlexanderWKoenig** [comment]
> @AlexanderWKoenig Hello and thank you for the report. Can you retest with Proton Experimental? We've made a fix that should hopefully have fixed it.

Yes it is fixed with latest proton experimental on my desktop arch linux. Great job! Will try it tomorrow on the steam deck.
Im trying to play through the campain and will report here if anything strange happens.

Big thanks and keep the great work rollin in :+1: 

EDIT: Played through the campain on Steam Deck, had no more issues and it worked really well.
